app/settings/surveys/survey-editor.directive.js

```diff
@@ -637,7 +637,7 @@ function SurveyEditorController(
         .saveCache(_.extend({ roles: $scope.roles_allowed }, { formId: $scope.survey.id }))
         .$promise
         .then(function (roles) {
-            $location.path('settings/surveys/edit/' + $scope.survey.id);
+            $location.url('/settings/surveys');
```

Comment:
Yea, I can work on that. Should I start a new branch or keep working with this one?